sql/promscale--0.1.1.sql

```diff
@@ -82,77 +83,78 @@ GRANT EXECUTE ON FUNCTION @extschema@.label_unnest(anyarray) TO prom_reader;
 
 ---------------------  comparison functions ---------------------
 
-CREATE OR REPLACE FUNCTION @extschema@.label_find_key_equal(key_to_match label_key, pat pattern)
+CREATE FUNCTION @extschema@.label_find_key_equal(key_to_match prom_api.label_key, pat prom_api.pattern)
 RETURNS matcher_positive
```

Comment:
doesn't `matcher_positive` also need to be schema qualified?

Reply:
hmm. yeah, I don't know whether unqualified return types are an attack vector but it seems like qualifying them would be good for consistency anyway

Reply:
Done.